### PR TITLE
Remove the serverLabels config if we don't set any labels

### PR DESCRIPTION
**Pull request by NotANormalNerd**

### CODE REVIEW

**Review by mxschmitt — COMMENTED** [review]
I think it would be enough if you would add it like we have with the networks to the validate method.

**Comment by NotANormalNerd** [comment]
Yes we could do that. I think it does not matter really, since we still have to delete the serverLabels. 
Since when it is available it will be given to the docker-machine-driver? (not entirely sure)

**Comment by mxschmitt** [comment]
> Yes we could do that. I think it does not matter really, since we still have to delete the serverLabels.
> Since when it is available it will be given to the docker-machine-driver? (not entirely sure)

I think its cleaner if we do it there, since I already do the same with the networks there. Since a few days afaik and I changed it in the README also since a few days.

**Comment by NotANormalNerd** [comment]
I changed it. You are right. Doing this in validation is cleaner then doing this in the setLabel action.

File: component/component.js [hunk 1]
```diff
@@ -67,6 +67,13 @@ export default Ember.Component.extend(NodeDriver, {
       this.set('model.%%DRIVERNAME%%Config.networks', [])
     }
 
+    let labels_list = this.get('model.%%DRIVERNAME%%Config.serverLabel');
```
Comment (mxschmitt) [inline component/component.js:70]:
```
    if (!this.get('model.%%DRIVERNAME%%Config.networks')) {
      this.set('model.%%DRIVERNAME%%Config.networks', [])
    }
```

shouldn't it be something like that enough? (did you test it?)

Comment (NotANormalNerd) [reply component/component.js]: Yes also works in my tests. 

I testes both. With and without labels. Actually created a server. No errors in any of the logs.

**Review by mxschmitt — APPROVED** [review]

**Comment by mxschmitt** [comment]
Thanks!

**Pull request merged**